kahuna/public/js/errors/global.html

```diff
@@ -42,6 +42,23 @@
                  ng-click="ctrl.dismiss('unknown')">close</gr-icon>
     </div>
 
+    <div class="global-error success autoHide" id="clipboard" ng-message="clipboard" tabindex="0" ng-blur="ctrl.dismiss('clipboard')">
+        <gr-icon >done</gr-icon>
+        A link to your images has been copied to your clipboard. Preview the URL
+        <a href="{{ctrl.errors['clipboard']}}"  rel="noopener" target="_blank" class="coloured-link">here</a>.
+
+        <gr-icon class="global-error__close"
+                 title="Close"
+                 ng-click="ctrl.dismiss('clipboard')">close</gr-icon>
+    </div>
+    <div class="global-error error autoHide" id="clipboard-max-warn" ng-message="clipboard-max-warn" tabindex="0" ng-blur="ctrl.dismiss('clipboard-max-warn')">
+        <gr-icon >warning</gr-icon>
+        You can only share a maximum of 44 images, please reduce the number of images in order to share your selection.
+        <gr-icon class="global-error__close"
+                 title="Close"
+                 ng-click="ctrl.dismiss('clipboard-max-warn')">close</gr-icon>
+    </div>
```

Comment:
Hi Andrew, apologies for the delays as I was facilitating a workshop the past two days. We came up with this full width pop up banner in order to capture our users attention in a much more accessible way and the wider accessibility aspect.  The banner is also consistent with our other internal tools at the BBC

Had conversations with the wider UX team around where it is known that disabled buttons are not being accessible hence for our full width banner design. Disabled buttons are designed in a way that they cannot be read by a screen reader (buttons are not focusable, and hence users can't reach them with a keyboard)

I've just seen the current tooltip for the first time today and it is very small. The gr-tooltip is a slight improvement. We could deploy that and have it as a question

Reply:
thanks for the context, it's very helpful. unfortunately there's a lot of users who do large selections of images without aiming to share - the popup will be extremely annoying for them. I'm sure there's a way we can be accessible without being annoying!
I had a quick google and came across this documentation for `aria-disabled` - do you think this would make the disabled buttons usable for screenreader users? https://a11y-101.com/development/aria-disabled
Otherwise, maybe we could leave the button not-disabled and show the error when they click share, rather than when they do the selection?

Reply:
Yep, banner must go for the above reason.

> could leave the button not-disabled

I would vote against as we have other disabled buttons, so it would be inconsistent.

Best would be to find a solution to screenreaders being silly about disabled buttons. Perhaps in the following PR.